<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>abs</class>
 <widget class="QWidget" name="abs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1134</width>
    <height>869</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Читательский дневник</string>
  </property>
  <widget class="QLineEdit" name="create_text">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>651</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>681</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Введите свои инициалы в формате &quot;Фамилия Имя Отчество&quot;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="create_1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>330</y>
     <width>241</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Создать</string>
   </property>
  </widget>
  <widget class="QLabel" name="create_ready">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>651</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="back_to_start">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>650</y>
     <width>241</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Вернуться на главную</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
